package.json

```diff
@@ -1189,17 +1189,17 @@
                     "when": "view == aws.explorer && viewItem == awsDocumentItemNodeWriteable"
                 },
                 {
-                    "command": "aws.command.runCommandInContainer",
+                    "command": "aws.ecs.runCommandInContainer",
```

Comment:
Why change the command title (same for the other commands)?

Reply:
Adding the qualifier to commands was requested by @justinmk3

Reply:
I suggested this because no other command is defined as `aws.command`. It is a new pattern. 

If we want to switch to that pattern then we will need to "namespace" command some other way, which means.... a prefix, like `ecs`.

Reply:
Ack, had it mentally reversed: we stick ".command." on the string name, not the command. Swap this feedback to that.